test/test_cpp/testcase.cpp

```diff
@@ -25,31 +25,30 @@ TestCase::~TestCase()
 
 void TestCase::checkDatabase_test()
 {
-    qDebug() << "Starting test of DatabaseEngine class's methods";
-    qDebug() << "Add new user Ivan";
-
-    User ivan("1234", "ivan");
-    DatabaseEngine db1;
-    db1.openDatabase(ivan);
-    QString filePath = QStandardPaths::writableLocation(QStandardPaths::AppLocalDataLocation) +
-            "/Database/" + ivan.id + ".db";
-
-    qDebug() << "Users Ivan have been registered. "
-             << QString("After %1 seconds user Ivan data will be showed on screen").arg(CHECK_TIME);
-    qDebug() << "Please, check database" << filePath;
-    QTest::qSleep(1000 * CHECK_TIME);
-
-    qDebug() << db1.getUser().name;
-
-    qDebug() << QString("After %1 seconds user Ivan will be removed!").arg(CHECK_TIME / 2);
-    qDebug() << "Please, check debug!";
-    QTest::qSleep(500 * CHECK_TIME);
-
-    db1.closeDatabase();
-    QFile::remove(filePath);
-
-    qDebug() << "Please, check debug! User have been removed.";
-    qDebug() << "End of DatabaseEngine class's methods test";
+    User ivan("1234", "ivan"), vova, dima, vlad;
+    dima.id = "5678";
+    vlad.name = "vlad";
+    QVector<User> vector;
+    vector.append(ivan);
```

Comment:
```suggestion
    vector << ivan << vova << dima << vlad;
```
Все же это тест и можно записать их покороче